api/app/models/FormValidator.scala

```diff
@@ -0,0 +1,22 @@
+package models
+
+import com.trifectalabs.roadquality.v0.models._
+
+object FormValidator {
+  def validateSegmentForm(segForm: SegmentForm): Seq[FormError] = {
+    { segForm.points.map { p =>
+        if (p.lat > 180 || p.lat < -180 || p.lng > 180 || p.lng < -180) Some(FormError(s"$p out of bounds"))
```

Comment:
Yes - this is just a sanity check to ensure that the points are valid (on earth). As we roll-out city/country support we can think through an implementation to ensure that segments are only being created in the correct locations though.